docker/PKGBUILD/kaldi.sh

```diff
@@ -0,0 +1,81 @@
+# Maintainer: Jingbei Li <i@jingbei.lli>
```

Comment:
These files are copied from AUR packages. (If you search for their names you can find them. I do have a list somewhere too.) Most PKGBUILD files needed to be modified in order for them to compile. I thought of a few possibilities:
1. Submitting patches for these AUR packages so they actually build correctly.
2. Run `sed` to correct the problems.
3. Store the files in the repository and copy them in to build.

I went with option three because it was clearer how the package was being built. You can see the script that is building the dependency and it keeps the docker file cleaner.

The packages need to be updated whenever it stops working. More to the point, I didn't substantially modify the PKGBUILD files so that you could run `diff` to see the changes between the latest AUR version our version. That makes upgrading slightly easier.